components/prism-llvm.js

```diff
@@ -0,0 +1,12 @@
+(function(Prism) {
+	Prism.languages.llvm = {
+		'comment': /;.*/,
+		'string': /"[^"]*"/,
+		'variable': /[%@!#](?:(?:[-a-zA-Z$._]|\\[a-fA-F0-9]{2})(?:[-a-zA-Z$._0-9]|\\[a-fA-F0-9]{2})*|[0-9]+)/,
+		'label': /(?:[-a-zA-Z$._]|\\[a-fA-F0-9]{2})(?:[-a-zA-Z$._0-9]|\\[a-fA-F0-9]{2})*:/,
+		'type': /\b(?:void|label|token|metadata|float|double|half)\b/,
+		'keyword': /[a-z_][a-z_0-9]*/,
```

Comment:
1. We need to checks for word boundaries or numbers like `0xabc` will be partially matched as `keyword`.
2. Is it really a good idea to highlight every literal we haven't matched just as a `keyword`? We often use long keyword lists for this to avoid false positives. Could this be applied here as well? (I'm asking because I have no idea about LLVM.)

Reply:
LLVM has thousands of keywords and doesn't make fine-grained stability promises, adding and removing them especially in extensions. Also, this looks like it's what they're doing in the syntax highlighting they use in their examples.

Reply:
Alright. That makes things simpler for us.
Thank you for sharing your insight.

```suggestion
		'keyword': /\b[a-z_][a-z_0-9]*\b/,
```